source/agora/node/FullNode.d

```diff
@@ -1026,7 +1043,12 @@ public class FullNode : API
         this.recordReq("postEnrollment");
 
         UTXO utxo;
-        this.utxo_set.peekUTXO(enroll.utxo_key, utxo);
+        if (!this.utxo_set.peekUTXO(enroll.utxo_key, utxo))
+        {
+            this.enroll_man.addMissingEnrollment(enroll.utxo_key);
```

Comment:
Wait that is not a "missing enrollment", it's a missing UTXO referenced in an enrollment

Reply:
I see what you mean. I will try to save missing `Enrollment`s instead of UTXOs.